pkg/email/system_intake_update_lcid.go

```diff
@@ -85,6 +92,7 @@ func (sie systemIntakeEmails) SendUpdateLCIDNotification(
 	ctx context.Context,
```

Comment:
Can you remove the 3319 TODO comments on these notification functions?

Reply:
Good catch, I meant to run back and do these after, lemme grab those now

Reply:
c8004bc9d